src/api/schemas/extendedQueryNode.graphql

```diff
@@ -209,7 +209,7 @@ input VideoWhereInput {
   createdAt_gte: Date
   isCurated_eq: Boolean
   isPublic_eq: Boolean
-  id_in: [ID!]
+  id_in: [ID]
```

Comment:
Why this change?

Reply:
for consistency as `channelId_in: [ID]` above didn't have the `!`

Reply:
They both should have it, check out https://hydra.joystream.org/graphql